CONFIGURATION.md

```diff
@@ -8,7 +8,6 @@ This document is generated from configuration structs in the source code using `
 |:---------------------|:-----|:--------|:------------|
 | YNABBER_DATADIR | `string` | `.` | DataDir is the path for storing files |
 | YNABBER_DEBUG | `bool` | `false` | Debug prints more log statements |
-| YNABBER_INTERVAL | `time.Duration` | `6h` | Interval is how often to execute the read/write loop, 0=run only once |
```

Comment:
@hpernu this will be a breaking change for you but its overdue that we move this value into the Nordigen lib.

Reply:
Sure, let me know when you want to merge. This should be a strict refactor and work both with interval at zero and non-zero. I tested it quite a few times, i'm also running this version myself.

Reply:
Sure. Hopefully soon. But, in the meanwhile, can you make a release with the current main branch? It is easier to check with that if I can actually make everything work with the mostly current version.

We can make a new release soon after.

+ I also created a new issue related to request/response logging for both Nordigen and YNAB. Your input is much appreciated.

Reply:
I am not actually using docker in the CI which is used to periodically pull the transactions although I am using docker on desktop. In CI, I am referring to tags to get a fixed version of ynabber. Version number are equivalent in docker and main branch tags. 

But you can do anything you want with this anyway. I am having trouble with the latest version regardless because the log format has changed although it seems to work. I am still figuring out the log thing.